src/pages/cart-page/Cart.jsx

```diff
@@ -1,17 +1,29 @@
-import React, { useContext, useState } from "react";
-import { Header } from "../../components";
+import axios from "axios";
+import React, { useContext, useEffect } from "react";
 import { WishlistContext } from "../../context";
 import { CartContext } from "../../context/cart-context";
 import "./Cart.css";
 
 export function Cart() {
   const { state, dispatch } = useContext(CartContext);
-  const [qty, setQty] = useState(1);
+  // const [qty, setQty] = useState(1);
+  // const encodedToken = localStorage.getItem("token");
+  // useEffect(() => {
+  //   (async () => {
+  //     try {
+  //       const result = await axios.get("/api/user/cart",  {headers: {
+  //         authorization: encodedToken,
+  //       },})
+  //       console.log(result)
+  //     } catch (err) {
+  //       console.log(err);
+  //     }
+  //   })();
+  // }, [encodedToken]);
```

Comment:
remove commented code if not going to use it

Reply:
sure